### ISSUE TITLE
logs display "INF Unable to sniff out cluster ID, perhaps set $AWS_CLUSTER_ID to force one"

### ISSUE THREAD

**Issue by ndallavalentina** [opencost/opencost#1753]
**Describe the bug**
Log error: 
```INF Unable to sniff out cluster-ID, perhaps set $AWS_CLUSTER_ID to force one```
It keeps appearing on the logs 

there is no info on the default installation about this environment variable:
https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/opencost.yaml

I guess if it's not set by default should be like AWS_CLUSTER_ID = CLUSTER_ID then we need just 1 env variable as both will be the same:
- here you can see that AWS_CLUSTER_ID will be equal to ClusterIdEnvVar 
https://github.com/opencost/opencost/blob/3507d95d08803ef541ce3f147b40db500d10c1b6/pkg/cloud/awsprovider.go#L300

- ClusterIdEnvVar is equal to CLUSTER_ID 
https://github.com/opencost/opencost/blob/3507d95d08803ef541ce3f147b40db500d10c1b6/pkg/env/costmodelenv.go#L25

I guess this is the code part affected:
https://github.com/opencost/opencost/blob/3507d95d08803ef541ce3f147b40db500d10c1b6/pkg/cloud/awsprovider.go#L1344

**To Reproduce**
Install Open cost check into the log error 

**Expected behavior**
N/A

**Screenshots**
N/A

**Which version of OpenCost are you using?**
Opencost v1.100

**Additional context**
N/A

**Comment by AjayTripathy** [comment]
@nikovacevic perhaps this log should be changed to WARN to allow users to filter it out?

**Comment by ndallavalentina** [comment]
maybe also change this line: https://github.com/opencost/opencost/blob/3507d95d08803ef541ce3f147b40db500d10c1b6/pkg/cloud/awsprovider.go#L1345
with: `defaultClusterName := env.ClusterIDEnvVar`

**Comment by AjayTripathy** [comment]
Great suggestion actually @nicolodallavalentinaclearscore ! @nikovacevic friendly ping? Maybe a quick one during our fix-it week for 102?

**Comment by Adam-Stack-PM** [comment]
https://kubecost.atlassian.net/browse/BURNDOWN-30

**Comment by nikovacevic** [comment]
@AjayTripathy yeah that's a fine suggestion

**Comment by ameijer** [comment]
@nikovacevic @AjayTripathy I took a shot at the implementation here: https://github.com/opencost/opencost/pull/1814. 

Looking at this AWS_CLUSTER_ID env var, I think we can deprecate it in the future - AWS is the only provider with this special cluster ID in use. To that end, I have added a deprecation warning to it, and maybe we can schedule the deprecation for a few releases from now? That should give people enough time to react to the deprecation warning and minimize compatibility issues if somebody is using AWS_CLUSTER_ID in their helm chart vars and such.
